proto/dydxprotocol/prices/streaming.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package dydxprotocol.prices;
+
+import "gogoproto/gogo.proto";
+import "dydxprotocol/prices/market_price.proto";
+
+option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/prices/types";
+
+// StreamPriceUpdate provides information on a price update.
+message StreamPriceUpdate {
```

Comment:
Do we need to add this to [StagedFinalizeBlockEvent](https://github.com/dydxprotocol/v4-chain/blob/80414ac20fa292af152d8ae69879d0a385288620/proto/dydxprotocol/clob/streaming.proto#L10)?

Reply:
Added!